Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -292,28 +292,26 @@ extension Tab {
         
         let group = DispatchGroup()
         group.enter()
-        
-        webView.evaluateSafeJavaScript(functionName: "document.documentElement.outerHTML.toString", contentWorld: .defaultClient) { html, _ in
+
+        webView.evaluateSafeJavaScript(functionName: "new XMLSerializer().serializeToString(document)", contentWorld: .defaultClient) { html, _ in
```

Comment:
Updated to match desktop

Reply:
It should be: `webView.evaluateSafeJavascript(functionName: "new XMLSerializer().serializeToString", args: ["document"], escapeArgs: false)`